VKontakte/Controller/Groups.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  Groups.swift
+//  VKontakte
+//
+//  Created by Ольга on 25.01.2023.
+//
+
+import UIKit
+
+final class Groups: UITableViewController {
```

Comment:
```suggestion
final class GroupsTableViewController: UITableViewController {
```

шо за плохие названия